setup.py

```diff
@@ -143,8 +147,12 @@ def run(self):
 
         # NB: Not a glob, because you can't build both onnx.proto and
         # onnx-proto.ml in the same build
-        proto_files = [os.path.join(SRC_DIR, "onnx.proto"),
-                       os.path.join(SRC_DIR, "onnx-operators.proto")]
+        if (1 == self.onnxml):
```

Comment:
ONNX_ML (and `self.onnxml`) is boolean, do `if ONNX_ML`

Reply:
fixed. Thanks!